logger_manager.py

```diff
@@ -0,0 +1,45 @@
+# Standard library imports
+import logging
+import logging.handlers
+import os
+
+
+def setup_logging(log_name, log_level=logging.INFO):
+    """ Setup logger.
+
+    :param log_name: name for logger
+    :param log_level: log level, a string
+    :return: a logger object
+    """
+
+    # Make path till log file
+    log_name += ".log"
+    log_file = os.path.join(os.getcwd(), log_name)
```

Comment:
don't use `os.getcwd()`, it's a moving target, use `os.path.dirname(__file__)`

Reply:
Done